erpnext/stock/doctype/repost_item_valuation/repost_item_valuation.py

```diff
@@ -309,6 +310,13 @@ def repost(doc):
 			frappe.db.commit()
 
 
+def remove_attached_file(docname):
+	if file_name := frappe.db.get_value(
+		"File", {"attached_to_name": docname, "attached_to_doctype": "Repost Item Valuation"}, "name"
+	):
+		frappe.delete_doc("File", file_name)
```

Comment:
```suggestion
		frappe.delete_doc("File", file_name, delete_permanently=True)
```

Files are always deleted permanently, so keeping the JSON of it around is useless.